Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -784,7 +784,7 @@ void UnSpawn(NetworkIdentity identity)
         {
             Guid assetId = identity.assetId;
             
-            identity.OnNetworkDestroy();
+            identity.OnNetworkDestroy.Invoke();
```

Comment:
Should this call identity.NetworkDestroy(). instead?

Reply:
Yes I thought I got all of those with a .Invoke() search. sorry fixed below.